Calq/Screens/ExamScreen/ExamScreen.swift

```diff
@@ -103,16 +103,20 @@ struct ExamView: View {
         .padding(.horizontal, 10)
         .padding(.vertical, 5)
         .onAppear {
-            subject = getExam(type)
-            sliderValue = (subject != nil) ? Float(Int(subject!.exampoints)) : 0
+            sliderValue = 0
+            if let subject = getExam(type) {
+                sliderValue = Float(Int(subject.exampoints))
+            }
```

Comment:
This is a big improvement to the previous to the previous code and a lot more readable. So thank you :+1:
Wouldn't it also be useful to add a debug log to catch instances of the `getExam()` returning `nil` as this should not happen?

Reply:
yes sounds usefull 👍